src/hooks/useLoginAndRequestToken.ts

```diff
@@ -13,7 +13,10 @@ export default function useLoginAndRequestToken() {
     const address = await login()
     if (!address) throw new Error('Failed to login')
 
+    // TODO: uncomment when push
```

Comment:
what's this todo for?

Reply:
I commented this line for testing only. I'll remove this TODO